packages/commercetools/theme/nuxt.config.js

```diff
@@ -81,6 +84,7 @@ export default {
     ['@vue-storefront/nuxt-theme'],
     project-only-end */
     ['@vue-storefront/commercetools/nuxt', {
+      useMiddleware: true,
```

Comment:
It should be true by default as disabling middleware is basically cutting you off from any extensibility and is just for hacking purposes